src/main/java/core/basesyntax/Mashine.java

```diff
@@ -0,0 +1,6 @@
+package core.basesyntax;
+
+public abstract class Mashine {

```

Comment:
The class name `Mashine` is likely a typo. It should be `Machine` according to the task description. Please correct this to ensure consistency and functionality.